lgsm/config-default/config-lgsm/pwserver/_default.cfg

```diff
@@ -0,0 +1,187 @@
+##################################
+######## Default Settings ########
+##################################
+# DO NOT EDIT, ANY CHANGES WILL BE OVERWRITTEN!
+# Copy settings from here and use them in either:
+# common.cfg - applies settings to every instance.
+# [instance].cfg - applies settings to a specific instance.
+
+#### Game Server Settings ####
+
+## Predefined Parameters | https://docs.linuxgsm.com/configuration/start-parameters
+ip="0.0.0.0"
+port="8211"
+players="32"
+serverpassword=""
+adminpassword=""
+servername="LinuxGSM Palworld Server"
+
+## Server Parameters | https://docs.linuxgsm.com/configuration/start-parameters#additional-parameters
+## Game Server Docs | https://satisfactory.fandom.com/wiki/Dedicated_servers
+startparameters="-port=${port} -players=${players} -useperfthreads -NoAsyncLoadingThread -UseMultithreadForDS -publicip ${ip} -publicport ${port} EpicApp=PalServer -servername=${servername} -serverpassword=${serverpassword} adminpassword=${adminpassword}"
```

Comment:
did you check if that some parmater work without `-` in front ? looks strange at first to me

Reply:
yeah looks like it should be `-EpicApp ` and `-adminpassword`

Reply:
yeah `-adminpassword` should be right, the `EpicApp` Settings should be okay, as it is...

Reply:
Looks like both work please add the - for consistancy